clusterutil/cluster_validation_config.go

```diff
@@ -8,20 +8,41 @@ import (
 )
 
 type ClusterValidationConfig struct {
-	Label           string                  `yaml:"label" category:"experimental"`
-	registeredFlags flagext.RegisteredFlags `yaml:"-"`
+	Label            string                  `yaml:"label" category:"experimental"`
+	AdditionalLabels flagext.StringSliceCSV  `yaml:"additional_labels" category:"experimental"`
+	registeredFlags  flagext.RegisteredFlags `yaml:"-"`
 }
 
 func (cfg *ClusterValidationConfig) RegisterFlagsWithPrefix(prefix string, f *flag.FlagSet) {
 	cfg.registeredFlags = flagext.TrackRegisteredFlags(prefix, f, func(prefix string, f *flag.FlagSet) {
-		f.StringVar(&cfg.Label, prefix+"label", "", "Optionally define the cluster validation label.")
+		f.StringVar(&cfg.Label, prefix+"label", "", "Primary cluster validation label.")
+		f.Var(&cfg.AdditionalLabels, prefix+"additional-labels", "Comma-separated list of additional cluster validation labels for server-side validation.")
 	})
 }
 
 func (cfg *ClusterValidationConfig) RegisteredFlags() flagext.RegisteredFlags {
 	return cfg.registeredFlags
 }
 
+// GetEffectiveLabels returns the effective cluster validation labels.
+// It combines the primary Label with any AdditionalLabels.
+// The primary Label is always first if present, followed by AdditionalLabels.
+func (cfg *ClusterValidationConfig) GetEffectiveLabels() []string {
+	var labels []string
+	if cfg.Label != "" {
+		labels = append(labels, cfg.Label)
+	}
+	labels = append(labels, cfg.AdditionalLabels...)
+	return labels
+}
+
+// Validate ensures the cluster validation configuration is valid.
+// Both Label and AdditionalLabels can be set together.
+func (cfg *ClusterValidationConfig) Validate() error {
+	// Both fields can be set - no validation errors
+	return nil
+}
```

Comment:
I believe this can be removed

Reply:
Or better yet, validate that "label" is set if additional labels is set. Doesn't make sense to set "additional" labels but not the main one